### PR TITLE
Add CI for cluster-api-provider-azure

### PR DESCRIPTION
**Pull request by enxebre**
Add CI for https://github.com/openshift/cluster-api-provider-azure/
- gofmt
- govet
- bazel-verify
- bazel-test (unit)
- bazel-build

### CODE REVIEW

**Comment by enxebre** [comment]
cc @phantom

File: ci-operator/config/openshift/cluster-api-provider-azure/openshift-cluster-api-provider-azure-master.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    from: base
+    to: cluster-api-provider-azure
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  unit:
```
Comment (paulfantom) [inline ci-operator/config/openshift/cluster-api-provider-azure/openshift-cluster-api-provider-azure-master.yaml]: this is unused here and can be deleted.

**Comment by droslean** [comment]
please generate the jobs by using the `ci-operator-prowgen`

**Comment by paulfantom** [comment]
@droslean could we have an approval?

Also who creates new image repos in quay? I am asking since after merging this PR we will need it.

**Comment by droslean** [comment]
> Also who creates new image repos in quay? I am asking since after merging this PR we will need it.
@smarterclayton ^

**Comment by smarterclayton** [comment]
I do, but we can only do it once an image has been built.


On Mar 14, 2019, at 8:51 AM, Nikolaos Moraitis <notifications@github.com>
wrote:

Also who creates new image repos in quay? I am asking since after merging
this PR we will need it.
@smarterclayton <https://github.com/smarterclayton> ^

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/release/pull/3101#issuecomment-472839106>,
or mute
the thread

**Comment by paulfantom** [comment]
@smarterclayton repository already executed a postsubmit hook and image should have been created: https://openshift-gce-devel.appspot.com/build/origin-ci-test/logs/branch-ci-openshift-cluster-api-provider-azure-master-images/1

Can we have an image named as "origin-azure-machine-controllers" to be consistent with origin-aws-machine-controllers and origin-libvirt-machine-controllers images?

**Comment by smarterclayton** [comment]
The way you control that is in your ci operator file.  The definition for
Azure should be consistent with AWS


On Mar 14, 2019, at 11:44 AM, Paweł Krupa <notifications@github.com> wrote:

@smarterclayton <https://github.com/smarterclayton> repository already
executed a postsubmit hook and image should have been created:
https://openshift-gce-devel.appspot.com/build/origin-ci-test/logs/branch-ci-openshift-cluster-api-provider-azure-master-images/1

Can we have an image named as "origin-azure-machine-controllers" to be
consistent with origin-aws-machine-controllers and
origin-libvirt-machine-controllers images?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/release/pull/3101#issuecomment-472927446>,
or mute
the thread

**Pull request merged**